package.json

```diff
@@ -146,7 +146,7 @@
     "minimatch": "^3.0.4",
     "mkdirp": "^0.5.0",
     "mocha": "^2.2.5",
-    "mocha-junit-reporter": "^1.17.0",
+    "mocha-junit-reporter": "^1.23.1",
```

Comment:
Should we also update the reporter for the extension tests as well?

Reply:
Probably, i'll do this first and see how it goes before i mess with anything else :D